lib/board.rb

```diff
@@ -3,80 +3,75 @@
 OFFSET = 1
 
 class Board
-    attr_reader :height, :width, :data
+  attr_reader :height, :width, :data
 
-    def initialize(args={})
-        @height = args.fetch(:height, 3)
-        @width = args.fetch(:width, 3)
-        @data = args.fetch(:data, Array.new(9, OPEN_SPACE))
-    end
+  def initialize(args = {})
+    @height = args.fetch(:height, 3)
+    @width = args.fetch(:width, 3)
+    @data = args.fetch(:data, Array.new(9, OPEN_SPACE))
```

Comment:
What if data was defined as say 16 but width and height weren't defined? This wouldn't work. As in, these defaults are relying on each other but don't really